### PR TITLE
add bungalows 21 and 22

### PR DESCRIPTION
**Pull request by LilShiningMan**
#### Summary
Content "Add Bungalows 21 and 22"

#### Purpose of change

add house spawn variety

#### Testing

linted and spawned in game

#### Additional 
bungalow21 g
context
bungalow21 r
bungalow22 g
bungalow22 r

### CODE REVIEW

**Review by FarkasKBence — COMMENTED** [review]
I don't have much to add aside from a few nitpicks:

File: data/json/mapgen/house/bungalow21.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        ".`==G#     E R|6  ff #%.",
+        ".`==K#sHHHsy R|y hffh#%.",
+        ".`==G#####oo#####oo###%.",
+        ".`````^..%%%%...%%%%....",
+        "........................"
+      ],
+      "terrain": {
+        "0": "t_linoleum_gray",
+        "=": "t_floor_noroof",
+        "?": "t_shrub_lilac",
```
[the remark this answers is not in the document]
Comment (LilShiningMan) [reply data/json/mapgen/house/bungalow21.json:39]: These are meant to be lilac bushes, not the "%" regional bushes in the house palette.

**Comment by John-Candlebury** [comment]
The bathroom in bungalow 21 probably deserves a window.

**Comment by LilShiningMan** [comment]
window added

**Comment by LilShiningMan** [comment]
I've also reverted the lab control for the bungalow21's garage back to the mechanical as it actually works.

**Comment by LovamKicsiGazsii** [comment]
> I've also reverted the lab control for the bungalow21's garage back to the mechanical as it actually works.

Gates have a random chance to fail, are you sure you're not mistaking it with that?

kép

Also, `t_gates_mech_control` doesn't even work with rock walls, so there's that.

**Comment by LilShiningMan** [comment]
ok, you convinced me, changed back. It turned out I did just have bad luck and it didn't open when I tried it.

**Pull request merged**